test/test_other.py

```diff
@@ -237,41 +237,57 @@ def test_emcc_generate_config(self):
         self.assertContained('LLVM_ROOT', config_contents)
         os.remove(config_path)
 
-  def test_emcc_output_mjs(self):
-    self.run_process([EMCC, '-o', 'hello_world.mjs', test_file('hello_world.c')])
-    output = read_file('hello_world.mjs')
-    self.assertContained('export default Module;', output)
-    # TODO(sbc): Test that this is actually runnable.  We currently don't have
-    # any tests for EXPORT_ES6 but once we do this should be enabled.
-    # self.assertContained('hello, world!', self.run_js('hello_world.mjs'))
+  @parameterized({
+    '': ([],),
+    'node': (['-sENVIRONMENT=node'],),
+  })
+  def test_emcc_output_mjs(self, args):
+    create_file('extern-post.js', 'await Module();')
+    self.run_process([EMCC, '-o', 'hello_world.mjs',
+                      '--extern-post-js', 'extern-post.js',
+                      test_file('hello_world.c')] + args)
+    src = read_file('hello_world.mjs')
+    self.assertContained('export default Module;', src)
+    self.assertContained('hello, world!', self.run_js('hello_world.mjs'))
 
   @parameterized({
     '': (True, [],),
     'no_import_meta': (False, ['-sUSE_ES6_IMPORT_META=0'],),
   })
+  @node_pthreads
```

Comment:
Why pthreads here?

Reply:
The test links with `-pthread`, so Node.js before version 16 needs the `--experimental-wasm-bulk-memory --experimental-wasm-threads` flags to successfully execute the `subdir/hello_world.mjs` script.